app/src/androidTest/java/com/arctouch/wanderley/italo/movieforyou/robots/MovieDetailAnalyserRobot.java

```diff
@@ -0,0 +1,43 @@
+package com.arctouch.wanderley.italo.movieforyou.robots;
+
+import com.arctouch.wanderley.italo.movieforyou.R;
+import com.arctouch.wanderley.italo.movieforyou.data.Movie;
+
+/**
+ * Created by italowanderley on 13/02/17.
+ */
+
+public class MovieDetailAnalyserRobot extends BaseViewAnalyserRobot {
+    private Movie mMovie;
```

Comment:
I am a 100% ok with this variable name, however the majority apple here in ArcTouch do not like. So just a heads up :)